src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -496,13 +440,6 @@ static ExchangeAlgorithmType MapExchangeAlgorithmType(const SSL_CIPHER* cipher)
 
         case SSL_KeyExchangeAlgorithm::SSL_kSRP:
             return ExchangeAlgorithmType::SSL_kSRP;
-#else
-        case SSL_KeyExchangeAlgorithm::SSL_kECDH:
-            return ExchangeAlgorithmType::SSL_ECDH;
-
-        case SSL_KeyExchangeAlgorithm::SSL_kECDHE:
-            return ExchangeAlgorithmType::SSL_ECDSA;
```

Comment:
The managed code is getting the cipher-suite information by calling `QueryContextConnectionInfo`. As far as I understand there is no adapter for Windows code.
We'd need some tests to verify the behavior.